localisation/000_wc_traits.csv

```diff
@@ -357,7 +357,7 @@ eyes_mutation;Multiple Eyes;Yeux multiples;Mehrere Augen;;;;;;;;;;;x
 eyes_mutation_desc;Under the influence of the Old Gods the body of this character has covered with frightening eyes.;;Unter dem Einfluss der Magie der alten G�tter hat sich der K�rper dieser Figur mit furchterregenden Augen bedeckt.;;;;;;;;;;;x
 creature_nraqi;N'raqi;N'raqi;N'raqi;;;;;;;;;;;x
 creature_nraqi_desc;This character is n'raqi. N'raqi, of the faceless ones, are a monstrous race that arose from the organic matter that seeped from the Old Gods' blighted forms during the primordial era of Azeroth. N'raqi are very insidious and cruel, their uncanny forms, resembling something between a mollusk and an insect, can plunge into shock every mortal creature.;;Dieser Charakter ist ein N'raqi. N'raqi, von den Gesichtslosen, sind eine monstr�se Rasse, die aus der organischen Substanz entstand, die w�hrend der Urzeit Azeroths aus den verdorbenen Formen der alten G�tter sickerte. N'raqi sind �u�erst heimt�ckisch und grausam, ihre unheimlichen Formen, die etwas zwischen einer Muschel und einem Insekt �hneln, k�nnen jede sterbliche Kreatur in Schock versetzen.;;;;;;;;;;;x
-creature_cthraxxi;C'thraxxi;C'thraxxi;C'thraxxi;;;;;;;;;;;x
+creature_cthraxxi;C'Thrax;C'thraxxi;C'thraxxi;;;;;;;;;;;x
```

Comment:
can you change it in all 3 languages pls?

Reply:
Done in 4746f97ff9c43457f23703dd05f9ac36153ddb04.